LoopKitUI/Views/ExpandablePicker.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  ExpandablePicker.swift
+//  LoopKitUI
+//
+//  Created by Anna Quinlan on 8/13/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import SwiftUI
+
+public struct ExpandablePicker: View {
+    @State var pickerShouldExpand = false
+    @State var pickerIndex: Int = 0
+    var onUpdate: (Int) -> Void
+    let label: String
+    
+    let items: [String]
+    
+    public init (
+        with items: [String],
+        onUpdate: @escaping (Int) -> Void,
+        label: String = ""
+    ) {
+        self.items = items
+        self.onUpdate = onUpdate
+        self.label = label
+    }
+    
+    public var body: some View {
+        VStack(alignment: .leading, spacing: 0) {
+            HStack(alignment: .center) {
+                Text(label)
+                Spacer()
+                Text(items[pickerIndex])
+                .foregroundColor(.gray)
+            }
+            .padding(.vertical, 5)
+            .frame(minWidth: 0, maxWidth: .infinity).onTapGesture {
+                self.pickerShouldExpand.toggle()
+            }
+            if pickerShouldExpand {
+                HStack {
+                    Picker(selection: $pickerIndex.onChange(onUpdate), label: Text("")) {
```

Comment:
This is a little bit hacky, but since bindings can't be passed into a child view and then into a picker, this is the best solution I was able to come up with